libs/mark-flow-ui/src/components/contest_header.tsx

```diff
@@ -48,10 +53,16 @@ export function Breadcrumbs(props: BreadcrumbMetadata): React.ReactNode {
 }
 
 export function ContestHeader(props: ContestHeaderProps): JSX.Element {
-  const { breadcrumbs, children, contest, district } = props;
+  const {
+    breadcrumbs,
+    children,
+    contest,
+    district,
+    styleOverrides = { horizontalPadding: true },
+  } = props;
```

Comment:
If you pass in a `className` prop and pass it to `Container`, you can use styled-components to add custom CSS styling and override the padding. I suggest since custom styling props can become a bit convoluted.

Alternatively I'd be into just removing the horizontal padding from this component and adding padding around it as needed at various render sites.

Reply:
Thanks! Ended up overriding with `className` to favor a smaller diff.